sdk/servicebus/service-bus/test/common/environmentVariables.ts

```diff
@@ -0,0 +1,106 @@
+const isNode = !!process && !!process.version && !!process.versions && !!process.versions.node;
+
+export enum EnvVarKeys {
+  SERVICEBUS_CONNECTION_STRING = "SERVICEBUS_CONNECTION_STRING",
+  QUEUE_NAME = "QUEUE_NAME",
+  QUEUE_NAME_NO_PARTITION = "QUEUE_NAME_NO_PARTITION",
+  QUEUE_NAME_SESSION = "QUEUE_NAME_SESSION",
+  QUEUE_NAME_NO_PARTITION_SESSION = "QUEUE_NAME_NO_PARTITION_SESSION",
+  TOPIC_NAME = "TOPIC_NAME",
+  TOPIC_NAME_NO_PARTITION = "TOPIC_NAME_NO_PARTITION",
+  TOPIC_NAME_SESSION = "TOPIC_NAME_SESSION",
+  TOPIC_NAME_NO_PARTITION_SESSION = "TOPIC_NAME_NO_PARTITION_SESSION",
+  SUBSCRIPTION_NAME = "SUBSCRIPTION_NAME",
+  SUBSCRIPTION_NAME_NO_PARTITION = "SUBSCRIPTION_NAME_NO_PARTITION",
+  SUBSCRIPTION_NAME_SESSION = "SUBSCRIPTION_NAME_SESSION",
+  SUBSCRIPTION_NAME_NO_PARTITION_SESSION = "SUBSCRIPTION_NAME_NO_PARTITION_SESSION",
+  TOPIC_FILTER_NAME = "TOPIC_FILTER_NAME",
+  TOPIC_FILTER_SUBSCRIPTION_NAME = "TOPIC_FILTER_SUBSCRIPTION_NAME",
+  TOPIC_FILTER_DEFAULT_SUBSCRIPTION_NAME = "TOPIC_FILTER_DEFAULT_SUBSCRIPTION_NAME",
+  AAD_CLIENT_ID = "AAD_CLIENT_ID",
+  AAD_CLIENT_SECRET = "AAD_CLIENT_SECRET",
+  AAD_TENANT_ID = "AAD_TENANT_ID",
+  RESOURCE_GROUP = "RESOURCE_GROUP",
+  AZURE_SUBSCRIPTION_ID = "AZURE_SUBSCRIPTION_ID",
+  CLEAN_NAMESPACE = "CLEAN_NAMESPACE"
+}
+
+const mandatoryEnvVars = [EnvVarKeys.SERVICEBUS_CONNECTION_STRING];
+
+const aadRelatedEnvVars = [
+  EnvVarKeys.AAD_CLIENT_ID,
+  EnvVarKeys.AAD_CLIENT_SECRET,
+  EnvVarKeys.AAD_TENANT_ID,
+  EnvVarKeys.AZURE_SUBSCRIPTION_ID,
+  EnvVarKeys.RESOURCE_GROUP
+];
+
+function throwErrorIfGivenEnvironmentVariablesAreMissing(envVars: EnvVarKeys[]): void {
```

Comment:
Function names usually don't refer to the input parameters. I would suggest `throwMissingEnvVarsError` or `throwMissingEnvironmentVariablesError`

Reply:
Will do